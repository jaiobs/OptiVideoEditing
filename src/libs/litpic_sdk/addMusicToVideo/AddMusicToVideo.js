import React, {Component} from 'react';
import {View, requireNativeComponent, findNodeHandle} from 'react-native';

const MusicToVideo = requireNativeComponent('MusicToVideo');

export default class AddMusicToVideo extends Component {
  constructor(props) {
    super(props);
  }

  _setReference = (ref: ?Object) => {
    if (ref) {
      this._AddMusicToVideoRef = ref;
      this._AddMusicToVideoHandle = findNodeHandle(ref);
    } else {
      this._AddMusicToVideoRef = null;
      this._AddMusicToVideoHandle = null;
    }
  };

  render() {
    return (
      <View style={{flex: 1}}>
        <MusicToVideo ref={this._setReference} {...this.props} />
      </View>
    );
  }
}
