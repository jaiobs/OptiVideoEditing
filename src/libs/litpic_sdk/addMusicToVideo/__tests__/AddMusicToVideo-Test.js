import React from 'react';
import renderer from 'react-test-renderer';
import AddMusicToVideo from '../AddMusicToVideo';

describe("AddMusicToVideo snapshot test",() => {
    it("It should render AddMusicToVideo Correctly", () =>{
        const tree = renderer.create(
            <AddMusicToVideo></AddMusicToVideo>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
});