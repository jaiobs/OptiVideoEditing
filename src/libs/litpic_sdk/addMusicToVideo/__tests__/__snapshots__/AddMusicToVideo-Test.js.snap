// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMusicToVideo snapshot test It should render AddMusicToVideo Correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <MusicToVideo />
</View>
`;
